src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -1,34 +1,25 @@
 package de.cas.mse.exercise;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> sumDigPow(long min, long max) {
 
-		List<Long> result = new ArrayList<Long>();
+		List<Long> result = new ArrayList<>();
 
-		for (long i = a; i < b; i++) {
+		for (long currentNumber = min; currentNumber < max; currentNumber++) {
 
-			List<Long> longs = new ArrayList<Long>();
+			List<Long> numbers = new ArrayList<>();
 
-			String temp = Long.toString(i);
+			splitValues(numbers, currentNumber);
```

Comment:
Statt Liste zu übergeben evtl. besser Liste als Rückgabewert

Reply:
+1 the naming of methods should make clear if passed parameters are changed inside the method